src/main/resources/lang/en.xml

```diff
@@ -366,4 +362,19 @@
         <snowflake>snowflake</snowflake>
         <snowflakes>snowflakes</snowflakes>
     </snowflakes>
+    <death>
```

Comment:
It definetly looks like a mess, but it's the order / paths that ocn uses, so it's the best way to implement that imo